### ISSUE TITLE
Article list wording revamp

### ISSUE THREAD

**Issue by MrOrz** [cofacts/rumors-line-bot#141]
From https://g0v.hackmd.io/CWKXj50SRQuCfxxlFi3STg#1-分享後查詢---有多則-or-不夠像時 , this provides reason why user needs to choose and motivate them to choose from existing messages in DB, instead of always creating new ones.

```
🔍 我們在資料庫裡，找到 3 則訊息，跟您傳給我的「⋯⋯」有點接近。

不實訊息常常會被人修改重發，
請選擇您想查看的訊息👇
```

**Comment by MrOrz** [comment]
https://developers.line.biz/console/fx-beta/

```
{
  "type": "flex",
  "altText": "Flex Message",
  "contents": {
    "type": "carousel",
    "contents": [
      {
        "type": "bubble",
        "header": {
          "type": "box",
          "layout": "horizontal",
          "contents": [
            {
              "type": "text",
              "text": "大部分文字相同",
              "margin": "none",
              "size": "sm",
              "weight": "bold",
              "color": "#AAAAAA"
            }
          ]
        },
        "body": {
          "type": "box",
          "layout": "horizontal",
          "spacing": "none",
          "margin": "none",
          "contents": [
            {
              "type": "text",
              "text": "暴徒的培訓及幕後的大台(一) :-  當大家都在想, 為何現在香港今天的局勢會搞成這樣?為何那些學生示威者會越來越激? 而且更像很有組織? 為何連警察及政府都無法招駕? 我今天為你們揭開真正的原因, 請大家務必要細看, 看後你們自會明白一切真相:-  我們先由2014年的佔中事件發生的兩年前開⋯⋯",
              "maxLines": 5,
              "flex": 0,
              "gravity": "top",
              "weight": "regular",
              "wrap": true
            }
          ]
        },
        "footer": {
          "type": "box",
          "layout": "horizontal",
          "contents": [
            {
              "type": "button",
              "action": {
                "type": "uri",
                "label": "選擇這篇",
                "uri": "https://linecorp.com"
              },
              "style": "primary"
            }
          ]
        }
      },
      {
        "type": "bubble",
        "direction": "ltr",
        "header": {
          "type": "box",
          "layout": "vertical",
          "contents": [
            {
              "type": "text",
              "text": "有些文字相同",
              "size": "sm",
              "align": "start",
              "color": "#AAAAAA"
            }
          ]
        },
        "body": {
          "type": "box",
          "layout": "vertical",
          "contents": [
            {
              "type": "text",
              "text": "我發現在世上原來有一個半公開的神秘的組織, 這個組織在香港及世界各地以”領英”這名義去物識一些有很多人物脈搏的人, 連我本人也曾被這個領英多之邀請加入, 但我當時也實在太忙了, 自己都⋯⋯",
              "maxLines": 5,
              "wrap": true
            }
          ]
        },
        "footer": {
          "type": "box",
          "layout": "horizontal",
          "contents": [
            {
              "type": "button",
              "action": {
                "type": "uri",
                "label": "選擇這篇",
                "uri": "https://linecorp.com"
              },
              "style": "primary"
            }
          ]
        }
      },
      {
        "type": "bubble",
        "hero": {
          "type": "image",
          "url": "https://via.placeholder.com/400x200?text=Q_Q",
          "size": "full",
          "aspectRatio": "20:13",
          "aspectMode": "cover",
          "action": {
            "type": "uri",
            "label": "Action",
            "uri": "https://linecorp.com/"
          }
        },
        "body": {
          "type": "box",
          "layout": "horizontal",
          "spacing": "md",
          "contents": [
            {
              "type": "button",
              "action": {
                "type": "uri",
                "label": "找不到我想查的訊息",
                "uri": "https://linecorp.com"
              },
              "style": "primary"
            }
          ]
        }
      }
    ]
  }
}
```

**Comment by MrOrz** [comment]
Implemented in #150
